accessor.go

```diff
@@ -54,28 +55,44 @@ func (sa *ShardAccessor) Shard() shard.Key {
 	return sa.shard.key
 }
 
-func (sa *ShardAccessor) Read(p []byte) (int, error) {
-	return sa.data.Read(p)
+// Reader returns an io.Reader that can be used to read the data from the shard.
+func (sa *ShardAccessor) Reader() io.Reader {
+	return &readerAtWrapper{
+		readerAt: sa.tryMmap(),
+	}
 }
 
 func (sa *ShardAccessor) Blockstore() (ReadBlockstore, error) {
-	var r io.ReaderAt = sa.data
+	r := sa.tryMmap()
```

Comment:
this doesn't need to be wrapped with the io.Reader wrapper?

Reply:
No, because the blockstore uses `ReaderAt` directly